.gitignore

```diff
@@ -31,3 +31,6 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+
+# Ignore simplecov coverage file.
+/coverage/
```

Comment:
We should ask about what to do to to get us back to 100 percent coverage on simple cov after seeing this addition did not help.